config/configmapprovider/retrieved_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package configmapprovider // import "go.opentelemetry.io/collector/config/configmapprovider"
```

Comment:
We add these annotations for usability automatically in all packages. Why removing here?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/blob/main/Makefile#L74-L76

Reply:
The tool adds only for non-test, check that almost all test don't have this, all other files have it.

Reply:
oh, I see. tests are skipped... nvm then